docs/Execution.md

```diff
@@ -29,52 +33,50 @@ QUIC_STATUS
     );
 ```
 
-Above is an example of a callback delivered to the listener interface.
-The application must register a per-object callback handler to manage all events MsQuic may indicate for that object, returning a status to show if it was successfully handled or not.
+The application must register a callback handler for every MsQuic object it creates. This handler must manage all the events MsQuic may indicate for that object. The handler must also return a status for each event indicating to MsQuic how the event was handled. This returned status is often success/failure, but sometimes indicates MsQuic that further processing is required.
```

Comment:
> for every MsQuic obejct it creates.

Some objects are created *by* MsQuic, but still need a callback handler. I'd reword this a bit.